nix/configuration.nix

```diff
@@ -91,6 +91,7 @@ in
       apply = x: {
         SITE_DOMAIN = if cfg.ssl then cfg.domain else "*";
         MEDIA_ROOT = "${cfg.dir}/media";
+        SENDFILE_ROOT = "/var/lib/concrexit/media";
```

Comment:
I think this folder is exactly the same one right? Would that not mean that the `SENDFILE_ROOT` should also be `${cfg.dir/media`?

Reply:
Yes